dmscripts/export_framework_applicant_details.py

```diff
@@ -212,8 +215,11 @@ def _format_field(field_name, field_value):
     return field_name, field_value
 
 
-def _create_row(framework_slug, record, count_statuses, framework_lot_slugs, dry_run=False):
+def _create_row(
+    framework_slug, record, count_statuses, framework_lot_slugs, dry_run=False, include_central_supplier_details=False
+):
     """
+    Fetch supplier data from central details, contact information and declaration
     :param dry_run:     if True return row without declaration information
```

Comment:
Could we have a line explaining what the new `include_central_supplier_details` param does?